lib/ui/platform_dispatcher.dart

```diff
@@ -213,6 +214,10 @@ class PlatformDispatcher {
         bottom: math.max(0.0, systemGestureInsetBottom),
         left: math.max(0.0, systemGestureInsetLeft),
       ),
+      // -1 is used as a sentinel for an undefined touch slop
+      gestureSettings: GestureSettings(
+        physicalTouchSlop: physicalTouchSlop < 0 ? null : physicalTouchSlop,
```

Comment:
does it need to be more precise? e.g. `physicalTouchSlop == undefinedSlope` or similar

Reply:
done